packages/amplify-cli/src/input-manager.ts

```diff
@@ -63,19 +64,47 @@ export function getCommandLineInput(pluginPlatform: PluginPlatform): Input {
   return result;
 }
 
+function preserveHelpInformation(input: Input): Input {
+  // preserve non-help command in subcommands
+  if (input.command && input.command.toLocaleLowerCase() !== constants.HELP) {
```

Comment:
What's the reasoning for using `toLocaleLowerCase` instead of `toLowerCase`?

Also one tiny nit, you can use [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) to simplify this slightly

```ts
  if (input?.command.toLocaleLowerCase() !== constants.HELP) {
```